+++
title = "Benchmarking Single-Cell RNA Sequencing Protocols for Cell Atlas Projects"
date = "2019-05-13"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.

authors = ["**Elisabetta Mereu**", "Atefeh Lafzi", "Catia Moutinho" , "Christoph Ziegenhain", "Davis J. MacCarthy",
"Adrian Alvarez","Eduard Batlle","Sagar","Dominic Grün","Julia K. Lau","Stéphane C. Boutet","Chad Sanada",
"Aik Ooi","Robert C. Jones","Kelly Kaihara","Chris Brampton","Yasha Talaga","Yohei Sasagawa","Kaori Tanaka",
"Tetsutaro Hayashi","Cornelius Fischer","Sascha Sauer","Timo Trefzer","Christian Conrad","Xian Adiconis","Lan T. Nguyen","Aviv Regev","Joshua Z. Levin","Swati Parekh","Aleksandar Janjic","Lucas E. Wange","Johannes W. Bagnoli",
"Wolfgang Enard","Marta Gut","Rickard Sandberg","Itoshi Nikaido","Ivo Gut","Oliver Stegle","Holger Heyn".]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference proceedings
# 2 = Journal
# 3 = Work in progress
# 4 = Technical report
# 5 = Book
# 6 = Book chapter
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "*Benchmarking Single-Cell RNA Sequencing Protocols for Cell Atlas Projects"
publication_short = "HCA Benchmarking"


# Abstract and optional shortened version.
abstract = "Single-cell RNA sequencing (scRNA-seq) is the leading technique for charting the molecular properties of individual cells. The latest methods are scalable to thousands of cells, enabling in-depth characterization of sample composition without prior knowledge. However, there are important differences between scRNA-seq techniques, and it remains unclear which are the most suitable protocols for drawing cell atlases of tissues, organs and organisms. We have generated benchmark datasets to systematically evaluate techniques in terms of their power to comprehensively describe cell types and states. We performed a multi-center study comparing 13 commonly used single-cell and single-nucleus RNA-seq protocols using a highly heterogeneous reference sample resource. Comparative and integrative analysis at cell type and state level revealed marked differences in protocol performance, highlighting a series of key features for cell atlas projects. These should be considered when defining guidelines and standards for international consortia, such as the Human Cell Atlas project."

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = true

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
#projects = ["example-external-project"]

# Links (optional).
url_pdf = ""
url_preprint = "https://doi.org/10.1101/630087"
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
[[url_custom]]
name = "Journal"
url = "https://doi.org/10.1101/630087"

# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = true
  
# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
#image = ""
#caption = "My caption :smile:"

+++


