+++
title = "Robustness and applicability of functional genomics tools on scRNA-seq data"
date = "2019-09-01"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.

authors = ["Christian H. Holland","Jovan Tanevski","Jan Gleixner","Manu P. Kumar","**Elisabetta Mereu**","Brian A. Joughin","Oliver Stegle","Douglas A. Lauffenburger","Holger Heyn","Bence Szalai","Julio Saez-Rodriguez"]



# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference proceedings
# 2 = Journal
# 3 = Work in progress
# 4 = Technical report
# 5 = Book
# 6 = Book chapter
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "*Robustness and applicability of functional genomics tools on scRNA-seq data"
publication_short = ""


# Abstract and optional shortened version.
abstract = "Many tools have been developed to extract functional and mechanistic insight from bulk transcriptome profiling data. With the advent of single-cell RNA sequencing (scRNA-seq), it is in principle possible to do such an analysis for single cells. However, scRNA-seq data has specific characteristics such as drop-out events, low library sizes and a comparatively large number of samples/cells. It is thus not clear if functional genomic tools established for bulk sequencing can be applied on scRNA-seq in a meaningful way. To address this question, we performed benchmark studies on in silico and in vitro single-cell RNA-seq data. We focused on the tools PROGENy and VIPER that estimate pathway and transcription factor (TF) activities, respectively. For the in silico study we simulated single cells from TF/pathway perturbation bulk RNA-seq experiments. Our simulation strategy guaranteies that the information of the original perturbation is preserved while resembling the characteristics of scRNA-seq data. We complemented the in silico data with in vitro scRNA-seq data upon CRISPR-mediated knock-out. Our benchmarks on both the simulated and in vitro data revealed comparable performance to the original bulk data. Additionally, we showed that the TF and pathways activities preserve cell-type specific variability by analysing a mixture sample sequenced with 13 scRNA-seq different protocols. Our analyses suggest that functional genomics tools can be used on scRNA-seq data, and provide a benchmark for further methods development by the community."

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = true

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
#projects = ["example-external-project"]

# Links (optional).
url_pdf = ""
url_preprint = "https://doi.org/10.1101/753319"
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
[[url_custom]]
name = "Journal"
url = "https://doi.org/10.1101/753319"

# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = true
  
# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
#image = ""
#caption = "My caption :smile:"

+++


